{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -216,8 +216,8 @@
 
 # ADMIN
 # ------------------------------------------------------------------------------
-# Django Admin URL regex.
-ADMIN_URL = r'^admin/'
+# Django Admin URL.
+ADMIN_URL = 'admin/'
```

Comment:
This pattern should be updated in -at least- 2 other places:
- In post gen hooks, for Docker: 
  https://github.com/pydanny/cookiecutter-django/blob/661fc3e449c779ad3fa2bad8964b6b7082f88ca3/hooks/post_gen_project.py#L159-L168
- In the documentation, for Heroku: http://cookiecutter-django.readthedocs.io/en/latest/deployment-on-heroku.html